api/src/main/java/com/upsidle/backend/persistent/domain/category/Category.java

```diff
@@ -0,0 +1,61 @@
+package com.upsidle.backend.persistent.domain.category;
+
+import com.upsidle.backend.persistent.domain.base.BaseEntity;
+import java.io.Serial;
+import java.io.Serializable;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * The product model for the application.
+ *
+ * @author Stephen Boakye
+ * @version 1.0
+ * @since 1.0
+ */
+@Entity
+@Getter
+@Setter
+@Table(name = "category")
```

Comment:
When the table name is not different from the class name, there is no need to use the @Table. Example, the User class already exists in many database engines so we rename it to @Table(name="users")